CHANGELOG.md

```diff
@@ -4,13 +4,13 @@ We follow the [Semantic Versioning 2.0.0](http://semver.org/) format.
 ## Unreleased
 
 ### Added
-- 
+-
 
 ### Changed
-- 
+- Fixed a bug where columns nested 2 levels deep in a large-gutter modifier gain the extra wide gutters.
```

Comment:
Stick `**cf-layout:** ` in front of the change to indicate the changed component (see [previous examples](https://raw.githubusercontent.com/cfpb/capital-framework/canary/CHANGELOG.md)).